src/index.md

```diff
@@ -4,8 +4,12 @@ heroText: Fault-Tolerant Stateful Code Platform
 tagline: Focus on your business logic and let Cadence take care of the complexity of distributed systems
 actionText: Get Started →
 actionLink: /docs/get-started/
-footer: © 2023 Uber Technologies, Inc.
+
 ---
+::: slot footer
```

Comment:
can you attach a before/after screenshot to the PR?

Reply:
sure, will get the screenshot of the change of the copyright year on the blog and main site.